File: pinyougou-interface/src/main/java/cn/itcast/core/service/SeckillService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.seckill.SeckillOrder;
<<<<<<< HEAD
import entity.PageResult;

public interface SeckillService {

    PageResult seckillorder(Integer page, Integer rows);
	
	    SeckillOrder findSeckillOrder(String name);

    void delSeckill(String name, Long seckllId);

}

File: pinyougou-web-user/src/main/webapp/js/service/seckillService.js
//服务层
app.service('seckillService',function($http){
	//读取列表数据绑定到表单中
	this.findSeckillOrder=function(){
		return $http.get('../seckill/findSeckillOrder.do');
	}
    this.delSeckill=function(seckillId){
        return $http.get('../seckill/delSeckill.do?id='+seckillId);
    }
});